### PR TITLE
Add a docs/ directory containing a mkdocs website

### PR DESCRIPTION
**Pull request by fbeutin-ledger**

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -0,0 +1,75 @@
+site_name: Ethereum plugins documentation
+site_url: https://LedgerHQ.github.io
+site_author: fbeutin@ledger.fr
```
Comment (cedelavergne-ledger) [inline mkdocs.yml:3]: SHould be better to have a generic email, like firrmware@... or embedded@... Don't know which ones are available

File: src/eth_plugin_interface.h [hunk 2]
```diff
@@ -55,167 +54,405 @@ typedef enum eth_plugin_result_e {
...
+The Ethereum application will call this handle when the plugin has requested
+ERC20 token information in `handle_finalize`
+
+The plugin needs to perform the following actions:
+
+- Check that provided token matches the one requested.
+- Increase the number of screens needed for the display if needed.
+- Report an error to the ethereum application if the display must be aborted.
 
+!!! warning
```
Comment (cedelavergne-ledger) [inline src/eth_plugin_interface.h:297]: The formating seems wrongly rendered.... Only the 1st line is in the box

File: mkdocs.yml [hunk 3]
```diff
@@ -0,0 +1,75 @@
+site_name: Ethereum plugins documentation
+site_url: https://LedgerHQ.github.io
```
[the remark this answers is not in the document]
Comment (fbeutin-ledger) [reply mkdocs.yml:2]: Changed to `https://ethereum-plugin-sdk.ledger.com`

File: src/eth_plugin_interface.h (hunk 2 above)
Comment (fbeutin-ledger) [reply src/eth_plugin_interface.h:297]: Fixed in this version

**Pull request merged**